src/scripts/OSFramework/OSUI/Behaviors/FocusTrap.ts

```diff
@@ -14,10 +15,11 @@ namespace OSFramework.OSUI.Behaviors {
 	 * @class FocusTrap
 	 */
 	export class FocusTrap {
+		private _canContainOtherPatterns = false;
 		private _firstFocusableElement: HTMLElement;
 		private _focusBottomCallback: GlobalCallbacks.Generic;
 		private _focusTopCallback: GlobalCallbacks.Generic;
-		private _focusableElements: HTMLElement[];
+		private _focusableElements: HTMLElement[] = [];
```

Comment:
Why are you initializing the array here, and not in the constructor?

Reply:
Done!